docs/dotnet-configuration.md

```diff
@@ -4,6 +4,91 @@ title: Scope .NET Agent configuration
 sidebar_label: Configuration
 ---
 
+The behaviour of the Scope Agent can be modified by using  environment variables or by using a configuration file `Scope.yml`in the source root of the project.
+
+> These configuration settings can also be provided via CLI flags. Run `scope-run --help` for more information.
+
+
+
+## Disabling Scope instrumentation
+
+You can disable Scope auto instrumentation. Scope auto instruments your code and tests by default, you can disable this behaviour.
+
+### Using Environment Variables
+
+```sh
+SCOPE_AUTO_INSTRUMENT=False
+```
+
+### Using YAML Configuration file
+
+```yaml
+scope:
+  auto_instrument: false
+  
+```
+
+
+
+## Changing service name
+
+You can change the name of the service when sending data to Scope. Scope initially uses `default` for the service name.
+
+### Using Environment Variables
+
+```sh
+SCOPE_SERVICE="service name"
+```
+
+### Using YAML Configuration file
+
+```yaml
+scope:
+  service: 'service name'
+```
+
+
+
+## Setting Scope as Global Tracer
+
+You can set Scope as the Global Tracer of Open Tracing. Scope uses Open Tracing internally to handle Span and traces, using its own tracer, You can set it as the global tracer so it will capture all the spans generated by any Open Tracing library in relation to your tests. 
```

Comment:
why is `Global Tracer` capitalised? 
Also, the phrase is a bit unclear imo.
How about something like
```suggestion
Scope handles Open Tracing span and traces using its own tracer. You can set Scope as the global tracer of Open Tracing so it will capture all the spans generated by any Open Tracing library in relation to your tests. 
```